app/assets/stylesheets/styles/game.scss

```diff
@@ -0,0 +1,53 @@
+/*// * {margin: 0; padding: 0; }
+// table { border-collapse: collapse; border-spacing: 0; }
+//
+// #chessboard { padding: 0px; margin: 0 auto; border: 2px solid #181818; }
+// #chessboard tr td { width: 60px; height: 60px; }
+// #chessboard tr:nth-child(2n) td:nth-child(2n+1) { background: #9f9f9f; }
+// #chessboard tr:nth-child(2n+1) td:nth-child(2n) { background: #9f9f9f; }*/
+
+#chessboard {
+  border-collapse: collapse;
+  margin: 0 auto .5em auto;
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Shorthand form for property `margin` should be written more concisely as `0 auto .5em` instead of `0 auto .5em auto`